helx/charts/roger/templates/webserver-configmap.yaml

```diff
@@ -0,0 +1,61 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: airflow-webserver-config
```

Comment:
This one is a bit tricky, this config is used https://github.com/helxplatform/devops/blob/1994c78a4fa30ab50c1c4d7bc629945cfd6f4fa6/helx/charts/roger/values.yaml#L103 in the values, where it would be mounted by airflow chart. 
That particular mount makes references to the webserver-configmap, But since we are only able to do such things (mounting secrets) to the airflow chart deployments through our values file, it makes it hard to resolve release dependent names from the values file (i.e i wasn't able to find a good way to make refs to the release name on the values file )

Reply:
Will have to think further on this one.  I did something similar with a configmap in the dug chart.

Reply:
just an FYI, There is a feature request for helm for such use cases https://github.com/helm/helm/pull/6876 , I have also tried playing with Yaml pointers (https://yaml.org/spec/1.2/spec.html#id2785586) but has some issues when parsed by helm where values are sometimes mis-assigned when overridden by --set